python/dgl/distributed/graph_services.py

```diff
@@ -81,7 +81,8 @@ def _sample_etype_neighbors(local_g, partition_book, seed_nodes, etype_field,
     local_ids = F.astype(local_ids, local_g.idtype)
     # local_ids = self.seed_nodes
     sampled_graph = local_sample_etype_neighbors(
-        local_g, local_ids, etype_field, fan_out, edge_dir, prob, replace, _dist_training=True)
+        local_g, local_ids, etype_field, fan_out, edge_dir, prob, replace,
+        etype_sorted=True, _dist_training=True)
```

Comment:
where is this variable defined?

Reply:
It is defined in dgl.sampling.neighbor.py L16

Reply:
ah, `etype_sorted` is the input argument name. i thought it's a variable in this function.

Reply:
is it guaranteed that the graph is sorted by etype in DistGraph?

Reply:
please write some comment that `DistGraph`'s edges are sorted.